src/test/ajaxWrapper.test.js

```diff
@@ -119,4 +133,64 @@ describe("ajaxWrapper", function () {
 
   });
 
+  describe("on POST request", function () {
+
+    it("sends the correct payload", function (done) {
+      var response = {body: ""};
+      var request = {method: null};
+      var payload = {"key": "value"};
+
+      this.server.on("request", function (req) {
```

Comment:
Yeah I do see the value in DRYing this up a notch. On the other hand I really like just exposing the `.on()` method as it's allows the greatest freedom.